docs/user_guides/fs/transformation_functions.md

```diff
@@ -0,0 +1,163 @@
+
+# Transformation Functions    
+
+In AI systems, [transformation functions](https://www.hopsworks.ai/dictionary/transformation) modify data used for machine learning applications, often to improve performance. The [taxonomy of data transformations](../../concepts/mlops/data_transformations.md) introduces three types of data transformation prevalent in all AI systems. Hopsworks offers simple Python APIs to define custom transformation functions. These can be used along with [feature groups](./feature_group/index.md) and [feature views](./feature_view/overview.md) to create [on-demand transformations](./feature_group/on_demand_transformations.md) and [model-dependent transformations](./feature_view/model-dependent-transformations.md), producing modular AI pipelines that are skew-free.
```

Comment:
modify data used for machine learning applications, often to improve performance ->
transform data to create features, the inputs to machine learning models (in both training and inference).